tests/foreman/ui_airgun/test_computeresource_ec2.py

```diff
@@ -0,0 +1,190 @@
+            'provider_content.secret_key': module_ec2_settings['secret_key'],
+            'provider_content.region.value': module_ec2_settings['region'],
+            'organizations.resources.assigned': [module_org.name],
+            'locations.resources.assigned': [module_loc.name],
+        })
+        cr_values = session.computeresource.read(cr_name)
+        assert cr_values['name'] == cr_name
+        assert cr_values['description'] == cr_description
+        assert (cr_values['organizations']['resources']['assigned']
+                == [module_org.name])
+        assert (cr_values['locations']['resources']['assigned']
+                == [module_loc.name])
+        session.computeresource.edit(cr_name, {
+            'name': new_cr_name,
+            'organizations.resources.assigned': [new_org.name],
+            'locations.resources.assigned': [new_loc.name],
+        })
+        assert not session.computeresource.search(cr_name)
+        cr_values = session.computeresource.read(new_cr_name)
+        assert cr_values['name'] == new_cr_name
+        assert (set(cr_values['organizations']['resources']['assigned'])
+                == {module_org.name, new_org.name})
+        assert (set(cr_values['locations']['resources']['assigned'])
+                == {module_loc.name, new_loc.name})
+        session.computeresource.update_computeprofile(
+            new_cr_name,
+            COMPUTE_PROFILE_LARGE,
+            {
+                'flavor': AWS_EC2_FLAVOR_T2_MICRO,
+                'availability_zone': module_ec2_settings['availability_zone'],
+                'subnet': module_ec2_settings['subnet'],
+                'security_groups.assigned': module_ec2_settings['security_groups'],
+                'managed_ip': module_ec2_settings['managed_ip'],
+            }
+        )
+        session.computeresource.delete(new_cr_name)
+        assert not session.computeresource.search(new_cr_name)
+
+
+@tier2
+def test_positive_create_ec2_with_custom_region(session, module_ec2_settings):
+    """Create a new ec2 compute resource with custom region
+
+    :id: aeb0c52e-34dd-4574-af34-a6d8721724a7
+
+    :customerscenario: true
+
+    :expectedresults: An ec2 compute resource is created
+        successfully.
+
+    :BZ: 1456942
+
+    :CaseLevel: Integration
+
+    :CaseImportance: Critical
+    """
+    cr_name = gen_string('alpha')
+    with session:
+        session.computeresource.create({
+            'name': cr_name,
+            'provider': FOREMAN_PROVIDERS['ec2'],
+            'provider_content.access_key': module_ec2_settings['access_key'],
+            'provider_content.secret_key': module_ec2_settings['secret_key'],
+            'provider_content.region.value': EC2_REGION_CA_CENTRAL_1
+        })
+        cr_values = session.computeresource.read(cr_name)
+        assert cr_values['name'] == cr_name
+
+
+@tier2
+def test_positive_create_with_default_profile(session, module_ec2_settings):
```

Comment:
that is redundant test case from our strategy point of view. If it has anything unique, add that to end to end scenario